python/oak/mapping_visu.py

```diff
@@ -12,6 +12,7 @@
 import threading
 import time
 import os
+from mapping_ar_renderers.invert4x4 import invert4x4
```

Comment:
Rather replace this with a function like this (assuming `a` is an `np.array`)
```python
def invert_se3(a):
    b = np.eye(4)
    b[:3, :3] = a[:3, :3].transpose()
    b[:3, 3] = -np.dot(b[:3, :3], a[:3, 3])
    return b
```
which is only valid for translation + rotation (that is, SE3) matrices, while your version works for generic 4x4 matrices